src/hamster-cli.py

```diff
@@ -141,6 +144,8 @@ def on_activate_quit(self, data=None):
 
     def on_startup(self, data=None):
         logger.debug("startup")
+        glib.set_application_name(self.__app_id__)
+        glib.set_prgname(self.__app_id__)
```

Comment:
Are both of those necessary?

Reply:
No, not to fix the icon name. But I think this way we're on the safe side.